src/Core/Configuration/AvifExtensionChecker.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
The namespace here is not correct, this configuration should be at the Image namespace level ie `PrestaShop\PrestaShop\Core\Image\`